showcase/app/templates/components/rich-tooltip.hbs

```diff
@@ -0,0 +1,1223 @@
+    <code>&lt;button&gt;</code>
+    and it doesn't have special styles of its own other than for the "focus" state.
+  </Shw::Text::Body>
+
+  {{#let (array "icon" "badge" "tag") as |toggles|}}
+    {{#each toggles as |toggle|}}
+      <Shw::Grid @columns={{4}} as |SG|>
+        {{#each @model.STATES as |state|}}
+          <SG.Item @label={{state}}>
+            <Hds::RichTooltip
+              @toggleAriaLabel={{if (eq toggle "icon") "Information"}}
+              mock-state-value={{state}}
+              mock-state-selector="button"
+              as |RT|
+            >
+              <RT.ToggleGeneric>
+                {{#if (eq toggle "icon")}}
+                  <FlightIcon @name="info" @isInlineBlock={{false}} />
+                {{/if}}
+                {{#if (eq toggle "badge")}}
+                  <Hds::Badge @text="Lorem ipsum" @color="neutral" @icon="hexagon" type="outlined" />
+                {{/if}}
+                {{#if (eq toggle "tag")}}
+                  <Hds::Tag @text="Lorem ipsum" />
+                {{/if}}
+              </RT.ToggleGeneric>
+              <RT.PopoverContent>
+                <Shw::Placeholder @text="generic content" @height="40" />
+              </RT.PopoverContent>
+            </Hds::RichTooltip>
+          </SG.Item>
+        {{/each}}
+      </Shw::Grid>
+    {{/each}}
+  {{/let}}
+
+  <Shw::Text::H4>With interactive content</Shw::Text::H4>
+
+  <Shw::Text::Body>
+    In this case the "toggle" wrapper is a simple
+    <code>&lt;div&gt;</code>
+    and it doesn't have any interaction states (the visual treatment of the inteactive states is the one of the
+    contained interactive element)
+  </Shw::Text::Body>
+
+  {{#let (array "link" "button") as |toggles|}}
+    {{#each toggles as |toggle|}}
+      <Shw::Grid @columns={{4}} as |SG|>
+        {{#each @model.STATES as |state|}}
+          <SG.Item @label={{state}}>
+            <Hds::RichTooltip
+              @toggleContainsInteractive={{true}}
+              mock-state-value={{state}}
+              mock-state-selector={{if (eq toggle "link") "a" "button"}}
+              as |RT|
+            >
+              <RT.ToggleGeneric>
+                {{#if (eq toggle "link")}}
+                  <Hds::Link::Standalone @icon="external-link" @iconPosition="trailing" @text="Acme Corp" @href="#" />
+                {{/if}}
+                {{#if (eq toggle "button")}}
+                  <Hds::Button @text="Lorem ipsum" @icon="wand" {{on "click" this.onClickButton}} />
+                {{/if}}
+              </RT.ToggleGeneric>
+              <RT.PopoverContent>
+                <Shw::Placeholder @text="generic content" @height="40" />
+              </RT.PopoverContent>
+            </Hds::RichTooltip>
+          </SG.Item>
+        {{/each}}
+      </Shw::Grid>
+    {{/each}}
+  {{/let}}
+
+  <Shw::Divider />
+
+  <Shw::Text::H2>Demo</Shw::Text::H2>
+
+  <Shw::Flex as |SF|>
+    <SF.Item @label="Inline with other text" {{style width="250px"}}>
```

Comment:
Question: I thought that if it's inline with other text, it should not have an icon? @jorytindall can you confirm?

Reply:
@MelSumner this is the showcase page. The design guidance may be different, but we have to make sure the component is resilient in case consumers use it in other ways (that may not be suggested, but still accessible/valid).

Reply:
I've added a recommendation for this 👍